src/components/ui/Dialog/stories/Dialog.stories.tsx

```
@@ -0,0 +1,55 @@
+};
+
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction#default-export
+export default {
+    title: 'WIP/Dialog',
+    component: Dialog,
+    render: (args:any) => {
+        return (
+            <SandboxEditor>
+                <Dialog.Root>
+                    <Dialog.Trigger >
+                        <>Open Dialog</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)